k8s/aat/common/traefik/traefik-neuvector-api-values.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: traefik-neuvector-api-values
```

Comment:
is this specific to the neuvector api at all?
could it just be traefik-cert-values

Reply:
The traefik instance is named traefik-neuvector-api.

Reply:
another traefik could use the cert though without having to store the cert in the repo twice